### DISCUSSION TITLE
ATP Synthase animation fails

### DISCUSSION THREAD

**Discussion by drquero** [BradyAJohnston/MolecularNodes#373] [Q&A] [answered]
Hi, 
I was trying to reproduce the ATP Synthase animation tutorial with molecular nodes. Still, when playing the animation, it doesn't move at all, even though I have all the frames. Do you have any idea what's wrong? Many thanks!
![Screenshot 2023-12-04 184431](https://github.com/BradyAJohnston/MolecularNodes/assets/152908146/dae9ad6d-caa3-4b07-9d3d-01ff04d377b2)

**Comment by BradyAJohnston** [comment]
Ah apologies, I have tweaked how the animation playback works but missed something. If you just change the `To Max` value to 61 in the `Animate Value` node, then it should playback. This is a bug and I will get a fix together soon!

**Reply by drquero** [reply]
Thank you very much for answering so fast! It worked perfectly! By the way, your add-on is fantastic (I will buy you a coffee soon to pay for the great job!)

**Comment by BradyAJohnston** [comment, marked as answer]
The import should now be fixed in the latest release: https://github.com/BradyAJohnston/MolecularNodes/releases
